code/modules/power/tesla/energy_ball.dm

```diff
@@ -144,6 +144,10 @@
 
 
 /obj/singularity/energy_ball/proc/dust_mobs(atom/A)
+	if(isliving(A))
```

Comment:
just saw that this was not really needed since it only works on carbon and the typecheck is already made below, but i guess it's not really a big deal